Dockerfile

```diff
@@ -0,0 +1,82 @@
+# syntax=docker/dockerfile:1
+
+ARG GO_VERSION=1.16.7
```

Comment:
(After this?) we should update to a current version of Go

Reply:
Yes just keeping minimal atm